datasources/04_ACT_Government.json

```diff
@@ -1,7 +1,15 @@
 {
   "catalog": [
     {
-      "items": [],
+      "items": [
+        {
+          "type": "socrata",
+          "name": "data ACT",
```

Comment:
Why not make the "ACT Government" folder itself query Socrata, rather than having a "data ACT" subfolder?